.rubocop.yml

```diff
@@ -1,3 +1,4 @@
+require: rubocop-rails
```

Comment:
Why? There's no rails in the project

Reply:
Yes we don't need this, but can you help why we have this in here: https://github.com/theforeman/smart_proxy_dynflow/blob/master/.rubocop.yml#L6-L7. I guess we can drop it.

Reply:
My guess would be the version we had previously was carrying rails-specific cops which were controlled by what you linked, but they have been moved into rubocop-rails since then.

No matter why, I'd remove this and the rails.enabled: false thing and we'll see what happens

Reply:
My approach is to use blame, which points to https://github.com/theforeman/smart_proxy_dynflow/commit/a4740fccacfcfdc92db5000d4e4aa9c777c1950b. As @adamruzicka suggests, predates the extraction of those cops to rubocop-rails. If you look at https://github.com/rubocop/rubocop-rails/blob/master/CHANGELOG.md#200-2019-05-22 then version 2.0.0 from 2019-05-22 is newer than that commit and has this line:
> Extract Rails cops from rubocop/rubocop repository

Reply:
@ekohl can you help me with this? it's the first time i find about blame.

Reply:
Thanks @ekohl!